package.json

```diff
@@ -4469,6 +4469,16 @@
 				"title": "Reset",
 				"category": "GitLens+"
 			},
+			{
+				"command": "gitlens.plus.resetRepositoryAccess",
+				"title": "Reset Repository Access",
+				"category": "GitLens+"
+			},
+			{
+				"command": "gitlens.plus.refreshRepositoryAccess",
+				"title": "Refresh Repository Access",
+				"category": "GitLens+"
+			},
```

Comment:
Should we call these "Reset All Repository Access" and "Reset Current Repository Access"?

Reply:
Or maybe "Reset Repository Access for All Repos" and "Reset Repository Access for Open Repos"?

Reply:
Yeah the wording on the second one sounds good ("for All Repos/for Open Repos"), as long as we are ok with the lengths.

Reply:
How about "Reset Repository Access Cache" (which aligns with the "Reset Avatar Cache") and "Refresh Repository Access" (since this isn't as much about a reset as a refresh for the user)

Reply:
Done in commit: [Update command wording](https://github.com/gitkraken/vscode-gitlens/pull/2510/commits/6322242bd3a3af52c1e60becb663ec122b67cd65)